python_modules/dagster-graphql/dagster_graphql/implementation/asset_checks_loader.py

```diff
@@ -0,0 +1,121 @@
+from typing import Iterable, List, Mapping, Optional
+
+from dagster import (
+    _check as check,
+)
+from dagster._core.definitions.events import AssetKey
+from dagster._core.definitions.external_asset_graph import ExternalAssetGraph
+from dagster._core.host_representation.external_data import ExternalAssetCheck  # noqa: TCH002
```

Comment:
seems like a new rule with ruff- since this is used only for type checks it wants to do it in a TYPE_CHECKING block https://docs.astral.sh/ruff/rules/typing-only-third-party-import/